.studio/automate-gateway

```diff
@@ -302,9 +291,11 @@ function send_chef_data_raw() {
   local token=$2
   if [[ "$1" == "legacy" ]]; then
     endpoint="${GATEWAY_URL}/api/v0/events/data-collector"
+  elif [[ "$1x" == "x" ]]; then
+    endpoint="${GATEWAY_URL}/api/v0/events/data-collector"
   elif [[ "$1" == "lb" ]]; then
     endpoint="https://localhost/data-collector/v0"
-  elif [[ "$1x" == "x" ]]; then
+  elif [[ "$1" == "run" ]]; then
```

Comment:
Changed the default to go through the data-collector because we are not using the direct run endpoint.